packages/blockchain/test/pos.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { Block } from '@ethereumjs/block'
+import Common, { Hardfork } from '@ethereumjs/common'
+import { BN } from 'ethereumjs-util'
+import tape from 'tape'
+import Blockchain from '../src'
+import testnet from './testdata/testnet.json'
+
+const buildChain = async (blockchain: Blockchain, common: Common, height: number) => {
+  const blocks: Block[] = []
+  const londonBlockNumber = common.hardforkBlockBN('london')!.toNumber()
+  const genesis = Block.genesis({}, { common })
+
+  const initialBaseFee = new BN(common.param('gasConfig', 'initialBaseFee'))
+  blocks.push(genesis)
+
+  for (let number = 1; number < height; number++) {
+    console.log(number)
+    let baseFeePerGas: BN = new BN(0);
+    if (number === londonBlockNumber) baseFeePerGas = new BN(10000)
```

Comment:
I wonder if we can make this more comfortable in the `Block` library, but I guess for now it's best to just manually set a hard-coded (so: not read with `Common.param()`) initial base fee to be set on the block, since the `Common` HF switch trigger hasn't occurred yet here.

I wonder though where you are getting this number from (`10000`), this would be `1000000000` (reading from the Common EIP definition.

Then the 1559 error goes away and this switches to a more descent error (`difficulty calculation is only supported on PoW chains`), which likely points to something not adopted in the Block or Blockchain library yet.

Reply:
This new error is then triggered by the `calcDifficultyByHeader` option for the `Block` package. I guess we want this option to just have no effect on non-PoW chains (of after PoW dismissing HFs 😋), I will directly push here with this change.

Reply:
Ok, did another push, tests should now pass. 😄

Reply:
Concur, thanks!